frontend/src/components/Table/ColumnFilter.tsx

```diff
@@ -21,7 +21,6 @@ const Wrapper = styled('div')`
   align-items: center;
   justify-items: center;
   position: relative;
-  font-size: 12px;
```

Comment:
This just keeps the text for the filterable columns the same size as the rest, other option is we reduce the size of the other ones to 12.